readthedocs/projects/static-src/projects/js/import.js

```diff
@@ -157,14 +157,24 @@ function ProjectImportView (instance, config) {
 
     ko.computed(function () {
         var org = self.filter_org(),
+            own = self.filter_own(),
             orgs = self.organizations(),
             url = self.page_current() || self.urls['remoterepository-list'];
 
-        if (org) {
-            url = append_url_params(
-                self.urls['remoterepository-list'],
-                {org: org}
-            );
+        if (!self.page_current()) {
```

Comment:
While working on this PR I found another issue:

1. go https://readthedocs.org/dashboard/import/
2. select an org to filter by
3. click next

you will note that the URL that is requested by AJAX never changes

I added this if here to fix that.